mcp/include/command_struct.h

```diff
@@ -294,10 +292,10 @@ typedef struct {
 	int new_cmd;
 	int reg;
 	float value;
-	uint16_t goal[N_AALBORG_VALVES];
-  	float dir[N_AALBORG_VALVES];
-  	float speed[N_AALBORG_VALVES];
-	int timeout[N_AALBORG_VALVES];
+	uint16_t goal[3];
+  	float dir[3];
+  	float speed[3];
+	int timeout[3];
```

Comment:
Why was the named constant replaced with "magic numbers"? Is it because the code defining it was moved to `legacyMCP`?

Reply:
Exactly correct. The aalborgs will be removed from reference entirely in the near future, but this was a way to temporarily keep them happy while I removed the module that defined the number.

Reply:
I'm in the middle of a large amount of excision so for some rabbit holes that I'm not entirely ready to go down, but I know will be removed I'll make a dependency removal change before the next sweep.

Reply:
Ah okay. If this code will be removed eventually, I won't be pestering you about bringing the named constant back. :wink: